platform/viewer/cypress/integration/common/OHIFStudyList.spec.js

```diff
@@ -0,0 +1,328 @@
+describe('OHIF Study List', function() {
+  context('Desktop resolution', function() {
+    beforeEach(function() {
+      cy.viewport(1750, 720);
+      cy.openStudyList();
+      cy.initStudyListAliasesOnDesktop();
+      // run these tests as if in a desktop
+    });
+
+    // //TO-TO: This test should be uncommented once issue #1114 is fixed: https://github.com/OHIF/Viewers/issues/1114
```

Comment:
I've responded to your issue with some additional context here:

- https://github.com/OHIF/Viewers/issues/1114#issuecomment-549216117

We may want to discuss how best to test given the new information, and if we should create new tickets/stories around improving the application to clear up any misconceptions users may have with search + search capabilities.

Reply:
Got it. In order to do a clean up here, I'll remove all the commented tests related to `fuzzymatching`. Once we decide to activate that, we can always go back and write one simple test to check it.